lxqt-base/lxqt-config/lxqt-config-0.13.0.ebuild

```diff
@@ -0,0 +1,53 @@
+# Copyright 1999-2018 Gentoo Foundation
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=6
+inherit cmake-utils versionator
+
+DESCRIPTION="LXQt system configuration control center"
+HOMEPAGE="http://lxqt.org/"
+
+if [[ ${PV} = *9999* ]]; then
+	inherit git-r3
+	EGIT_REPO_URI="https://github.com/lxqt/${PN}.git"
+else
+	SRC_URI="https://downloads.lxqt.org/downloads/${PN}/${PV}/${P}.tar.xz"
+	KEYWORDS="~amd64 ~arm ~arm64 ~x86"
+fi
+
+LICENSE="GPL-2 LGPL-2.1+"
+SLOT="0/$(get_version_component_range 2)"
+
+RDEPEND="
+	dev-libs/libqtxdg:0/3
+	dev-qt/qtconcurrent:5=
```

Comment:
Also, dev-qt/qtconcurrent can be dropped with: https://github.com/lxqt/lxqt-config/pull/233